lib/charms/karma_k8s/v0/karma_dashboard.py

```diff
@@ -37,7 +37,10 @@ def __init__(self, *args):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 1
```

Comment:
Current patch version on CharmHub is already 3, is this correct?

Reply:
Correct.

Reply:
if it's already 3, shouldn't this be 4?

Reply:
This is the alertmanager charm, it does not own the karma lib.
This change is after `fetch-lib`, so yes, it matches charmhub.